react-select/react-select-tests.tsx

```diff
@@ -61,6 +63,30 @@ const CustomValue = React.createClass({
     }
 });
 
+const filterOptions = (options: Array<Option>, filter: string, values: Array<Option>) => {
+    // Filter already selected values
+    let filteredOptions = options.filter(option => {
+        return !(_.includes(values, option));
```

Comment:
and while we're at it, do you really need `_.includes` or is `values.indexOf(option) < 0` sufficient?